src/pages/Team.js

```diff
@@ -10,25 +10,23 @@ function Team() {
   const teamMembers = t("teamMembers", { returnObjects: true });
 
   return (
-    <div>
+    <GridContainer>
+      <LanguageButtons page={t("team.pathname")} />
       <h1>{t("team.header")}</h1>
-      <GridContainer>
-        <CardGroup row gap={3}>
-          {teamMembers.map((person) => {
-            return (
-              <Grid desktop={{ col: 4 }} tablet={{ col: 6 }}>
-                <BioCard
-                  name={person.name}
-                  photo={person.photo}
-                  role={person.role}
-                />
-              </Grid>
-            );
-          })}
-        </CardGroup>
-        <LanguageButtons page={t("team.pathname")} />
-      </GridContainer>
-    </div>
+      <CardGroup row gap={3}>
+        {teamMembers.map((person) => {
+          return (
+            <Grid desktop={{ col: 4 }} tablet={{ col: 6 }}>
```

Comment:
This is pretty minor, but you can use the gridLayout attribute in your Card component and set your columns there, instead of having a Grid wrap the BioCard. Not sure if it would make much of a difference, but it is one less nested component!